pyked/chemked.py

```diff
@@ -646,11 +656,11 @@ def __init__(self, properties):
                 else:
                     raise ValueError('uncertainty-type must be one of "absolute" or "relative"')
 
-            composition[idx]['amount'] = quant
-        setattr(self, 'composition', composition)
+            #composition[idx]['amount'] = quant
 
-        self.equivalence_ratio = properties.get('equivalence-ratio')
+            setattr(self, 'composition', composition)
```

Comment:
This should only be set once, not on every time through the for loop (I think your indentation is a bit off)